src/Widgets/DataFormContainerWidget/DataFormContainerWidgetComponent.tsx

```diff
@@ -44,6 +44,9 @@ provideComponent(DataFormContainerWidget, ({ widget }) => {
         onSubmit={onSubmit}
         onReset={onReset}
         className={isSubmitting ? 'form-loading' : ''}
+        onClick={(e) => {
```

Comment:
* [ ] Not sure which use case this is for? If we want to prevent button clicks from propagating we may be better off with stopping the propagation on the button. Stopping the propagation on the form may lead to visitor and editor confusion:
  * Have a link with a headline and a form: a click on the headline **follows** the link
  * Have a link with a form that starts with a headline: a click on the headline **doesn't follow** the link
  * Both cases look identical to the visitor and more or less identical to the editor.

Reply:
I'll revert this from this PR, since it is no longer needed for events. Will resubmit as an individual PR, so we can discuss there.

Reply:
=> https://github.com/Scrivito/scrivito-portal-app/pull/338